### ISSUE TITLE
Upgrade Django to v5

### ISSUE THREAD

**Issue by saket-shukla** [Assystant/SpotAxis#1]
1. Upgrade django dependency in the project from 2.x to 5.x
2. Subsequently upgrade the python version from 2.x to 3.x
3. Handle code changes caused by python migration
4. Updates specific to an imported library can be ignored, however python/django specific imports and syntax updates need to be handled

**Comment by iayushrana** [comment]
yes, i have updated it before or recently in one of my project explain a little bit more about it.

**Comment by saket-shukla** [comment]
@iayushrana The current project is setup to use python 2.7 and Django 1.9.5
We need to upgrade django to the latest LTS supported version and the maximum supported python version for it
The requirement is only to update:
1. Requirements
2. Code relevant changes stemming from this upgrade
3. Package upgrades and their relevant changes are not expected from this upgrade, they will be handled via other tickets.

Let me know if you require any other information.

Please let us know if you will be picking this issue and we will assign it to you.

**Comment by Pranav-1717** [comment]
I want to work on it.

**Comment by saket-shukla** [comment]
Hi @Pranav-1717 
Sure, I am assigning this to you.
Please feel free to reach out if you have any questions or difficulties.

**Comment by Chintankansal48** [comment]
I want to work on this .

**Comment by saket-shukla** [comment]
@Chintankansal48 Assigned
